sources/models/src/lib.rs

```diff
@@ -127,6 +128,10 @@ struct KubernetesSettings {
     authentication_mode: KubernetesAuthenticationMode,
     bootstrap_token: KubernetesBootstrapToken,
     standalone_mode: bool,
+    eviction_hard: HashMap<EvictionHardKey, SingleLineString>,
+    kube_reserved: HashMap<ReservedResourcesKey, SingleLineString>,
```

Comment:
Can you add a comment above these two fields explaining why we're not doing strict validation on the values here? Something similar to https://github.com/bottlerocket-os/bottlerocket/blob/6b21873dd575cebe05148a6814059fa75df1ea35/sources/models/src/modeled_types/kubernetes.rs#L212-L217 
or
https://github.com/bottlerocket-os/bottlerocket/blob/6b21873dd575cebe05148a6814059fa75df1ea35/sources/models/src/lib.rs#L189